roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -56,8 +57,17 @@ default_backend = {{ cinder.default_backend }}
 
 {% for backend in cinder.backends %}
 [{{ backend.name }}]
-volume_driver = {{ backend.volume_driver }}
 volume_backend_name = {{ backend.name }}
+volume_driver = {{ backend.volume_driver }}
+{% if backend.name == "RBD_VOLUMES" %}
+rbd_pool = {{ ceph.rbd_pool }}
+rbd_user = {{ ceph.rbd_user }}
+rbd_secret_uuid = {{ ceph_uuid_file.content | b64decode }}
+rbd_ceph_conf = {{ ceph.rbd_ceph_conf }}
+rbd_flatten_volume_from_snapshot = {{ ceph.rbd_flatten_volume_from_snapshot }}
+rbd_max_clone_depth = {{ ceph.rbd_max_clone_depth }}
+glance_api_version = {{ ceph.glance_api_version }}
```

Comment:
I think because it wasn't in the config before these changes. But maybe it needs to be?